### ISSUE TITLE
ANSIBLE0013 to a shell script contains multiple lines

### ISSUE THREAD

**Issue by no2a** [willthames/ansible-lint#311]
# Issue Type
- Bug report

# Ansible and Ansible Lint details

```
ansible 2.4.2.0
ansible-lint 3.4.19
```

- ansible installation method: pip
- ansible-lint installation method: pip

# Desired Behaviour

Please give some details of the feature being requested
or what should happen if providing a bug report

with a playbook (x.yml)
```
- hosts: all
  gather_facts: no
  tasks:
  - name: get something done
    shell: |
      do_something
      touch /.done
    args:
      creates: /.done
    become: yes
```

Then, nothing should be warned. As the task has multiple commands with LF, using shell module is right:
```
$ ansible-lint x.yml 
(no output)
```

# Actual Behaviour (Bug report only)

I got ANSIBLE0013 as if LF is not contained:
```
$ ansible-lint x.yml 
[ANSIBLE0013] Use shell only when shell functionality is required
x.yml:4
Task/Handler: get something done
```

In my investigation, ```task["action"].get("__ansible_arguments__", [])``` was ```[u'do_something', u'touch', u'/.done']```. LF was removed from the input.
(at https://github.com/willthames/ansible-lint/blob/v3.4.19/lib/ansiblelint/rules/UseCommandInsteadOfShellRule.py#L44)

**Comment by willthames** [comment]
Thanks for this comprehensive bug report. I'm not sure why the LF gets stripped after parsing the YAML block but that's probably the place to start.

**Comment by willthames** [comment]
Looks like normalise_task_v2 uses `split()`, splitting on newlines and spaces, making it impossible to see if there were ever newlines.

If I change it to `split(' ')`, that should retain the desired behaviour (the main use for the split is to determine the command being run for other modules)

**Comment by willthames** [comment]
Fix pushed as v3.4.20
